src/lava/proc/conv/utils.py

```diff
@@ -0,0 +1,236 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import numpy as np
+from scipy import signal
+
+try:
+    import torch
+    import torch.nn.functional as F
+    TORCH_IS_AVAILABLE = True
+except ModuleNotFoundError:
+    TORCH_IS_AVAILABLE = False
```

Comment:
Yeah, torch implementation was just a test. We should definitely do something with tensorflow as well. The reason I didn't do it now is because grouping implementation needs work in tensorflow to align with similar interface. But yes, that's definitely a TODO.

The reason I didn't put the broadcast in utils is it will likely not be useful elsewhere except the initializaion. Plus it will be local here.